### PR TITLE
ENCD-5629-add-file-output-type-entries-to-glossary

### PR DESCRIPTION
**Pull request by iyoungworth**
minor-corrections

### CODE REVIEW

**Review by gabdank — DISMISSED** [review]

**Review by gabdank — CHANGES_REQUESTED** [review]
I think the updated description of alignments, should be applied also for unfiltered and transcriptome alignments.

**Comment by iyoungworth** [comment]
Thank you for noticing that, I updated the unfiltered and transcriptome alignments definitions and rebased

File: src/encoded/static/components/glossary/glossary.json [hunk 1]
```diff
@@ -1,10 +1,512 @@
 [
     {
-		"term": "Functional Genomics data",
+		"term": "functional genomics data",
 		"definition": "Data generated by assays investigating processes such as transcription, translation and epigenetic regulation on a genome-wide scale. Examples of assays generating functional genomics data: RNA-seq, ChIP-seq, DNase-seq, ATAC-seq, WGBS, HiC, and ChIA-PET."
 	},
 	{
-		"term": "Functional Characterization data",
+		"term": "functional characterization data",
 		"definition": "Data generated by assays (i.e STARR-seq, MPRA, and CRISPR screen) investigating the relationship between DNA sequences and their regulatory activities."
```
Comment (gabdank) [inline src/encoded/static/components/glossary/glossary.json]: @iyoungworth can  you please align these two old entries with the rest of the entries in the file? It looks like the indentation of these entries is different.

**Review by gabdank — CHANGES_REQUESTED** [review]
please correct the indentation

File: src/encoded/static/components/glossary/glossary.json (hunk 1 above)
Comment (iyoungworth) [reply src/encoded/static/components/glossary/glossary.json]: Sure thing- it looks like my text editor switched between tabs and spaces. They should all be consistent now.

**Review by gabdank — APPROVED** [review]

**Pull request merged**